src/task8.py

```diff
@@ -0,0 +1,62 @@
+def first_word(text: str) -> str:

```

Comment:
i don't know what is in this file because there are no task descriptions. 
nevertheless I don't see advanced tasks here (looking at solutions)